src/jit/gentree.cpp

```diff
@@ -334,6 +334,10 @@ void GenTree::InitNodeSize()
 #endif
 #endif // FEATURE_PUT_STRUCT_ARG_STK
 
+#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)
+    GenTree::s_gtNodeSizes[GT_HWIntrinsic]      = TREE_NODE_SZ_LARGE;
```

Comment:
Why is this a large node? It appears to have the same fields as `GT_SIMD`/`GenTreeSIMD`, which is a small node.

Reply:
My mistake, will fix.